.github/workflows/actionlint.yaml

```
@@ -0,0 +1,15 @@
+name: "Lint GitHub Actions Workflows"
+on: [push, pull_request]
```

Suggest only on push to main
```suggestion
on: [push, pull_request]
```